docs/sponsors.rst

```diff
@@ -0,0 +1,20 @@
+********
+Sponsors
+********
+
```

Comment:
Can you add a one-line explanation, something like "We're grateful to the following organisations for sponsoring us and helping us sustain TARDIS as a free and open source software."?